Wikipedia/Code/UserHistoryFunnel.swift

```diff
@@ -53,9 +53,35 @@ private typealias ContentGroupKindAndLoggingCode = (kind: WMFContentGroupKind, l
             event["test_group"] = articleAsLivingDocBucket.rawValue
         }
         
+        var autorizationStatus: UNAuthorizationStatus = .notDetermined
+        dataStore.notificationsController.notificationPermissionsStatus { status in
```

Comment:
I noticed the event always seems to post `notDetermined`, even if I have enabled notifications permissions (which you can enable via app Settings > Notifications > flip the toggle).

`dataStore.notificationsControllernotificationPermissionStatus` is asynchronous so it's probably not returning in time to update the `authorizationStatus` var before it gets added to the event dictionary. 

Instead I'd first try determining the status outside of the `event` method, then passing it in as a parameter once we definitely know we have it. Here's an example of what the `logStartingSnapshot()` event call would look like:

```
dataStore.notificationsController.notificationPermissionsStatus {[weak self] authorizationStatus in
            
    guard let self = self else {
        return
    }
            
    DispatchQueue.main.async {
        self.log(self.event(authorizationStatus: authorizationStatus))
    }
}
```